docker/docker-compose.yaml

```
@@ -0,0 +1,92 @@
+version: '3.8'
+
+# The services below should only be used for development and testing purposes. They are not production-ready.
+services:
+  jenkins-master:
+    build:
+      context: ./docker/master-node
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+      - "50000:50000"
+      - "5055:5055"
+    environment:
+      GITHUB_SSH_KEY: $GITHUB_SSH_KEY
+      GITHUB_SSH_KEY_PASSPHRASE: $GITHUB_SSH_KEY_PASSPHRASE
+      JENKINS_AGENT_NAMES: "agent-dotnet agent-python agent-js"
+      DATADOG_JENKINS_PLUGIN_REPORT_WITH: DSD
+      DATADOG_JENKINS_PLUGIN_COLLECT_BUILD_LOGS: false
+      DATADOG_JENKINS_PLUGIN_TARGET_HOST: datadog # `dogstatsd` or `datadog` based on the container you wish to use
+      DATADOG_JENKINS_PLUGIN_TARGET_LOG_COLLECTION_PORT: 10518
+      DATADOG_JENKINS_PLUGIN_TARGET_API_KEY: $JENKINS_PLUGIN_DATADOG_API_KEY
+    volumes:
+      - jenkins_shared:/var/jenkins_home/shared
+      - $JENKINS_PLUGIN/target/datadog.hpi:/var/jenkins_home/plugins/datadog.hpi
+
+  jenkins-agent-dotnet:
+    build:
+      context: ./docker/agent-node
```

```suggestion
      context: ./agent-node
```
Same as previous comment. Paths need to be relative to the `docker-compose.yaml`